app/views/schools/advice/electricity_intraday/_insights.html.erb

```diff
@@ -0,0 +1,50 @@
+<%= component 'notice', status: :neutral do |c| %>
+  <%= t('advice_pages.electricity_intraday.insights.summary') %>
+<% end %>
+
+<!-- Your current peak electricity use section -->
+<%= render 'schools/advice/section_title', section_id: 'your_current_peak_electricity_use', section_title: 'Your current peak electricity use' %>
+
+<table class="mt-2 table table-sm advice-table">
+  <thead>
+    <th class="text-left">
+      <%= t('advice_pages.electricity_intraday.insights.your_current_peak_electricity_use.table.average_peak_kw') %>
+    </th>
+    <th class="text-right">
+      <%= t('advice_pages.electricity_intraday.insights.your_current_peak_electricity_use.table.percent_change') %>
+    </th>
+  </thead>
+  <tr>
+    <td class="text-left">
+      <%= format_unit(@peak_usage_calculation.average_peak_kw, :kw) %>
+    </td>
+    <td class="text-right">
+      <%= format_unit(@peak_kw_usage_percentage_change, :percent) %>
```

Comment:
Wrap this in the `up_downify` helper (see baseload and long term usage insights for examples). This will give the coloured up and down arrows. And check whether we've been able to calculate